packages/graph/hash_graph/lib/graph/Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "HASH Graph API"
 
 [dependencies]
 async-trait = "0.1.59"
-axum = "0.5.16"
+axum = "0.6.1"
```

Comment:
(comment not for line)
Want to point out that there is a different URL routing behavior in 0.6 (PR1118 in the Axum repo, not linking to not cross-reference their PR). I don't think this affects us, but we have to keep in mind that this may be the cause of any routing issues if we do run into them